syntax = "proto2";

message HostPortProto {
  required string host = 1;
  required int32 port = 2;
}

message ClusterProto {
  repeated HostPortProto host_port = 1;
}
